_data/vendors.yml

```diff
@@ -115,6 +115,14 @@
   pricing_source: https://www.figma.com/pricing
   updated_at: 2019-10-19
 
+- name: Front
+  url: https://frontapp.com
+  base_pricing: $9 per u/m
```

Comment:
Honestly, the base pricing should probably be for the $25 tier, as that's the first tier at which you can have unlimited users. This would bring the percentage increase to 216%.